rllib/algorithms/simple_q/simple_q.py

```diff
@@ -55,8 +55,8 @@ class SimpleQConfig(AlgorithmConfig):
         >>>         "capacity":  40000,
         >>>     }
         >>> )
-        >>> config.training(replay_buffer_config=replay_config)\
-        ...       .resources(num_gpus=1)\
+        >>> config.training(replay_buffer_config=replay_config)
```

Comment:
when should we use >>> vs code blocks?

Reply:
See https://docs.ray.io/en/master/ray-contribute/writing-code-snippets.html#which-type-of-example-should-you-write.

Reply:
tl;dr: Never use `code-block:: python`. If you really want to emphasize object representations, use `>>>`. Most of the time you probably want `testcode`, though.